go/libkb/resolve.go

```diff
@@ -171,6 +171,7 @@ func (r *Resolver) resolveURLViaServerLookup(au AssertionURL, input string, with
 
 	var them *jsonw.Wrapper
 	if them, res.err = ares.Body.AtKey("them").ToArray(); res.err != nil {
+		res.err = NotFoundError{fmt.Sprintf("User not found for query: %q", input)}
```

Comment:
Then there will be the DELETED status code returned by user/lookup that we should use instead of looking at json.